### PR TITLE
data: sourav091

### PR DESCRIPTION
**Pull request by sourav091**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4803"></a>

### CODE REVIEW

**Review by rahulpandey70 — CHANGES_REQUESTED** [review]
Hey @sourav091 Please move your json file inside data folder.

File: sourav091.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "name": "Twitter: Threads and Random Talks",
+      "url": "https://twitter.com/Smandaltwts",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/souravmandal091/",
+      "icon": "FaLinkedin"
+    },
```
Comment (kumarsonsoff3) [inline sourav091.json:24]: There is an extra comma `,` and missing bracket `]`.

```suggestion
    } 
 ]
```

File: sourav091.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Sourav Mandal",
+  
+  "type": "personal",
+  "bio": "Python Development | Open Source Enthusiast | Putting My Work Here",
+  "tags": ["Open Source", "Android", "Flutter", "Python"],
+  ""
```
Comment (kumarsonsoff3) [inline sourav091.json:7]: If there is nothing, then please remove these `""`

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please move your file into the data directory, and to do so, you need to rename your **[file here](https://github.com/sourav091/LinkFree/edit/sourav091-patch-1/sourav091.json)** to `data/sourav091.json`

Change `sourav091.json`  - >  `data/sourav091.json` here. 

Let us know if you need any help with this! 

image

**Pull request merged**